src/Symfony/Bridge/Doctrine/phpunit.xml.dist

```diff
@@ -33,7 +33,12 @@
         <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener">
             <arguments>
                 <array>
-                    <element key="time-sensitive"><string>Symfony\Bridge\Doctrine\Middleware\Debug</string></element>
+                    <element key="time-sensitive">
+                        <array>
+                            <element key="0"><string>Symfony\Bridge\Doctrine\Middleware\Debug</string></element>
+                            <element key="1"><string>Symfony\Bridge\Doctrine\Middleware\Debug</string></element>
```

Comment:
@nicolas-grekas Isn't `Symfony\Bridge\Doctrine\Middleware\IdleConnection`?

Reply:
Oops, PR welcome

Reply:
fixed in 65ccca0758